pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -1,8 +1,11 @@
 { stdenv
 , lib
-, cudatoolkit
+, cudatoolkit ? null
+, libcublas ? null
+, zlib ? null
```

Comment:
Required regardless of which cudatoolkit distribution: cudnn on master _is_ broken

Reply:
Interesting, I thought ldd showed all was good before.